sink-compose.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
```

Comment:
Again, I prefer if we separate this to at least two PRs. I think the HAPI sink related changes are fine to be checked in this repo. but I am not sure about the atomfeed related ones.